torch/onnx/__init__.py

```diff
@@ -288,8 +288,10 @@ def forward(self, x):
             If a custom opset is referenced by ``model`` but not mentioned in this dictionary,
             the opset version is set to 1.
 
-        enable_onnx_checker (bool, default True): If True the onnx model checker will be run
-            to ensure the exported model is a valid ONNX model.
+        enable_onnx_checker (bool, default True): [Deprecated and ignored. Will be removed in next release]
```

Comment:
I'm not sure by the meaning of next release. Will we need to update this statement for every new release?
Maybe we should mention the release ID of when it was discontinued?

Reply:
I meant PyTorch release. I guess we can write "PyTorch release" in the docstring if that's clearer.

Reply:
My current plan is: after this change has been released in a new PyTorch release, then I will submit another PR to remove them from the public torch.onnx.export() function, and update these comments accordingly. So the release ID is not so important.

Reply:
Ideally, . As long as 

> I meant PyTorch release. I guess we can write "PyTorch release" in the docstring if that's clearer.

Users who can see these comments are trying to call torch.onnx.export() method, so I believe they will only be aware of PyTorch release. What's the other potential candidate?

Reply:
@garymm 

Users who see these comments are supposed to call torch.onnx.export() method, so they should only be aware of PyTorch release.

Are there any other possible candidates might confuse users?

Reply:
Clearly Shubham was confused, so please write "PyTorch release" to make it clearer.